### ISSUE TITLE
🏗 File structure

### ISSUE THREAD

**Issue by astroash** [fac-17/My-Body-Back#117]
Think about your file structure, and best ways to group similar items together. Currently your components folder is home to both reused components and page components. This makes it difficult to navigate and find specific files.

**Comment by mkatenolan** [comment]
Hey @astroash do you mean having two separate folders within components - one for individual pages and one for re-used components(ie. menu & header)?

**Comment by astroash** [comment]
Yeah, that would solve the problem. There's no definitive way of defining a file structure, it's more about thinking about how you can group things together to be self descriptive for easy access
